lib/frequency_table.rb

```diff
@@ -0,0 +1,44 @@
+# frozen_string_literal: true
+
+require "json"
+
+require "calculate_format"
+require "overlap/ht_item_overlap"
+
+class FrequencyTable
+  def initialize
+    @table = Hash.new do |hash, member|
+      hash[member] = Hash.new { |fmt_hash, fmt| fmt_hash[fmt] = Hash.new(0) }
+    end
+  end
+
+  def organizations
+    @table.keys.sort
+  end
+
+  def [](organization)
+    @table[organization.to_sym]
+  end
+
+  # For testing. I don't know if this is useful in the long run.
+  def to_h
```

Comment:
This was added just so I could easily integrate this class into `CostReport` specs which were expecting a Hash.